code/__DEFINES/preferences.dm

```diff
@@ -29,7 +29,6 @@
 #define UI_DARKMODE 131072
 #define DISABLE_KARMA 262144
 #define CHAT_NO_MENTORTICKETLOGS 524288
-
```

Comment:
What did you even do here

Reply:
this is outdated, I added in two preferences for AZERTY mode and numpad targetting since